src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -98,6 +98,7 @@
          DelaySign="$(DelaySign)"
          DisabledWarnings="$(NoWarn)"
          DocumentationFile="@(DocFileItem)"
+         EmbedAllSources="$(EmbedAllSources)"
```

Comment:
I believe it should say EmbedAllSource or EmbedAllSourceFiles.

Reply:
Yeah, ```EmbedAllSourceFiles``` would be probably somewhat better but F# already uses ```EmbedAllSources``` and it's good to be consistent across the board.

Reply:
(F# could introduce an alternative property, but then that would just add complexity that's not worth it).